website/pages/docs/builders/proxmox.mdx

```diff
@@ -251,6 +251,50 @@ builder.
 - `vm_interface` - (string) - Name of the network interface that Packer gets
   the VMs IP from. Defaults to the first non loopback interface.
 
+## Boot Command
```

Comment:
The updated documentation looks great. But it looks like there was an issue with the front matter for the Proxmox documentation - https://deploy-preview-9943--packer-www.netlify.app/docs/builders/proxmox
![image](https://user-images.githubusercontent.com/1749304/93251262-f6a33180-f761-11ea-814c-3d512f748582.png)

I pushed the commit to fix this issue since I was already looking into what happened.